content/en/blog/news/first-post/index.md

```diff
@@ -1,17 +1,131 @@
+
+But what if you mostly rely on images provided by your cloud-provider? Or deploy to environments that are not easily reproducible?
+
+This is where [runtime-class-manager][runtime-class-manager] (coming soon, formerly [KWasm][kwasm]) comes into play - a Production Ready and Kubernetes-native way to manage WebAssembly runtimes on Kubernetes Hosts. With runtime-class-manager you can ensure that containerd-shim-spin is installed on your hosts, manage version lifecycles, and security patches - all from the Kubernetes API.
+
+By combining these technologies, scaling a WebAssembly application looks more like this:
+
+| Phase | Description |
+| --- | --- |
+| Kube init | Kubernetes schedules a Pod to a node, the node then prepares to run the Pod |
+| Image pull (cacheable) | The node pulls the needed container images from a remote repository to a local cache when needed |
+| WASM Load (cachable) | The shim loads the webassembly and prepares it for execution on the node’s CPU architecture |
+| Application start | The shim starts listening on the configured port and is ready to serve |
+| Pod ready | The Pod responds to the asynchronous readiness probe and signals that it is now ready |
+| Service ready | The Pod gets added to the list of active endpoints for a Service and can start receiving traffic |
+
+Or in practice:
+
+<video src="spinkube-scaling.mp4" width="80%" controls></video>
+
+
+## Deploying applications
+
+This leaves one major area that can be painful when deploying applications to Kubernetes today: the developer and deployment experience.
+
+This is where [Spin][spin], [the `spin kube` plugin](https://github.com/spinkube/spin-plugin-kube), and [`spin-operator`][spin-operator] start to shine.
+
+![Architecture diagram of the SpinKube project](spinkube-diagram.png)
+
+The spin-operator makes managing serverless applications easy - you provide the
+image and bindings to secrets, and the operator handles realizing that
+configuration as regular Kubernetes objects.
+
+Combining the operator with the `spin kube` plugin gives you superpowers, as it
+streamlines the process of generating Kubernetes YAML for your application and
+gives you a powerful starting point for applications.
+
+This is all it takes to get a HTTP application running on Kubernetes (after
+installing the containerd shim and the operator):
+
+```bash
+# Create a new Spin App
+spin new -t http-rust --accept-defaults spin-kube-app
+cd spin-kube-app
+
+# Build the Spin App
+spin build
+
+# Push the Spin App to an OCI registry
+export IMAGE_NAME=ttl.sh/spin-app-$(uuidgen):1h
+spin registry push $IMAGE_NAME
+
+# Scaffold Kubernetes manifests
+spin kube scaffold -f $IMAGE_NAME > app.yaml
+
+# Deploy to Kubernetes
+kubectl apply -f app.yaml
+```
+
+If you want to try things out yourself, the easiest way is to follow the [quickstart guide][quickstart] - we can’t wait to see what you build.
+
+[spin-operator]: https://github.com/spinkube/spin-operator
```

Comment:
Please note: The double spaces at the end of each line (in this reference section) must stay so that each individual reference (URL) is on its own new line.

![Screenshot 2024-03-13 at 11 09 14](https://github.com/spinkube/documentation/assets/9831342/a295c11b-c421-4ab2-84ee-087c3fe7fa6c)

Reply:
these aren't supposed to show on the page - they're markdown link references, not content 😅